src/components/MapMarker.js

```diff
@@ -26,7 +26,7 @@ export default class MapMarker extends Component<Props> {
           <Container>
             <Row>
               <Col>
-                <img alt={berth.name.fi} width="180" src={berth.image} />
+                <img alt={berth.name.fi} width="180px" src={berth.image} />
```

Comment:
I don't think this is valid for IMG tag. https://html.spec.whatwg.org/#attr-dim-width